v1/styles.css

```diff
@@ -407,6 +426,12 @@ ul {
   padding: 0;
 }
 
+.section-header__item-right {
+  text-align: right;
+  padding-right: 16px;
+  color: var(--grey100);
```

Comment:
I included it in item-right, because the color is different from the left item, but I don't have a specific class for the left item.

Reply:
Yeah I understand that you want the text to be grey. This styling should not be in class called `item-right`. Only styling that makes the item right should go into this class. Please fix.

Reply:
I dont want `color: var(--grey100)` in a generic reusable class. Again if the styling isn't being reused it should have a class specific to that item. Having a class change a div's text color, that is supposed to ONLY MOVE THE ITEM RIGHT is quite confusing. Either rename it to specific class name to that item or move the color to something else like `.text-grey300` and add that class in the HTML. 

RULE OF THUMB - _Stop overthinking stuff_. Only add a generic class name if that specific styling will get reused. Otherwise name it something specific to that html element.

Reply:
If I understand correctly that is what I was trying to do, I named it item--right because it was the item on the right. It has the text-align: right because it was on the left side of the right half it that makes sense. I could also change it to .section-header__item > div if you don't like the class name would that be better?